lib/internal/cluster/round_robin_handle.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 const {
+  Array,
   Boolean,
-  Map,
+  Map
```

Comment:
```suggestion
  Map,
```

Sorry for the nit. Can you please keep the trailing comma?

Reply:
Done 👍